src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/TypesTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+using Microsoft.Spark.Sql.Types;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.IpcTests
+{
+    [Collection("Spark E2E Tests")]
+    public class TypesTests
+    {
+        private static IJvmBridge Jvm { get; } = SparkEnvironment.JvmBridge;
```

Comment:
@imback82 would it be better to use the SparkEnvironment to get the JvmBridge or the SparkFixture ?

Reply:
you mean define `Jvm` in `SparkFixutre` first then retrieve?

Reply:
Something like
```csharp
public TypesTests(SparkFixture fixture)
{
    _jvm = ((IJvmObjectReferenceProvider)fixture.Spark).Reference.Jvm;
    ...
}

Reply:
Or do the above to define it in the `SparkFixture`.

Reply:
If we are doing this, we want to define it in the `SparkFixture`.

Reply:
So would you suggest to use SparkEnvironment or SparkFigure? Thanks.

Reply:
I would say go ahead and use SparkFixture, adding the JVM property to it.

Reply:
Sure, I have made the changes. Let me know if that makes sense to you. Thanks.